reactor-netty-http/src/main/java/reactor/netty/http/server/HttpServerConfig.java

```diff
@@ -802,12 +819,18 @@ else if (metricsRecorder instanceof ContextAwareHttpServerMetricsRecorder) {
 				}
 			}
 		}
+
+		if (errorLogEnabled) {
+			p.addBefore(NettyPipeline.ReactiveBridge, NettyPipeline.ErrorLogHandler, new DefaultErrorLogHandler(errorLog));
+		}
```

Comment:
Register the `ErrorLogHandler` before the `ReactiveBridge` in HTTP/1.1 protocol..